simple/tests/stats/stat_var_hierarchy_generator_test.py

```diff
@@ -12,20 +12,81 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import json
```

Comment:
I've not read the tests yet, and maybe you're doing it already - should we replicate the tests we have from g3 here?

Reply:
It's in the spirit of the g3 tests but haven't replicated them here because not all features are implemented here yet. Let me take a look and see if some of them can be brought over though. If you have some in mind, let me know as well.

Reply:
Lets support MCF as input and add in these tests?

https://source.corp.google.com/piper///depot/google3/datacommons/prophet/flume_generator/test_data/svg/;bpv=0

Reply:
Follow up PR is fine.

Reply:
Ack.